hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/metadata/AbstractDatanodeDBDefinition.java

```diff
@@ -33,12 +33,16 @@
 
   private File dbDir;
 
+  private ConfigurationSource config;
+
   /**
    * @param dbPath The absolute path to the .db file corresponding to this
    * {@link DBDefinition}.
    */
-  protected AbstractDatanodeDBDefinition(String dbPath) {
+  protected AbstractDatanodeDBDefinition(String dbPath,
```

Comment:
Please update the java doc for the new parameter

Reply:
Sure.